Sofa/Component/Topology/Container/Dynamic/src/sofa/component/topology/container/dynamic/EdgeSetGeometryAlgorithms.inl

```diff
@@ -749,6 +750,7 @@ void EdgeSetGeometryAlgorithms<DataTypes>::draw(const core::visual::VisualParams
         vparams->drawTool()->drawPoints(positions, 4.0f, _drawColor.getValue());
     }
 
+    vparams->drawTool()->restoreLastState();
```

Comment:
just for the record, could you explain what was going wrong without restoring the last State?